### PR TITLE
Add helpers for turning content loader errors into govuk errors

### PR DESCRIPTION
**Pull request by lfdebrux**
Ticket: https://trello.com/c/xdGjiDqF/160-2-replace-validation-banner-with-govuk-frontend-error-summary-component-in-admin-frontend

---

## Outstanding:

- Tests
- Is this the right place for the module?

### CODE REVIEW

**Comment by risicle** [comment]
> Is this the right place for the module?

I think the "right place" for it TBH is `digitalmarketplace-govuk-frontend` but seeing as that's not a python package it would be a nightmare to import from there, this is probably the best place for it.

**Comment by risicle** [comment]
I've pushed a bunch of changes, see whatya think.

File: dmutils/forms/errors.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+        {{ forms[question.type](question, service_data, errors) }}
+
+    :param errors: A digitalmarketplace-frontend-toolkit error dictionary
+    :return: A dict with error information in a form suitable for
+             Digital Marketplace templates
+    """
+    return OrderedDict(
+        (key, {**error, **govuk_error(error)}) for key, error in errors.items()
+    )
```
[the remark this answers is not in the document]
Comment (risicle) [reply dmutils/forms/errors.py]: Well, we could import it there (or here) as an alias, to allow for an easy switchover. Provided it didn't create cyclical imports.

[the remark this answers is not in the document]
Comment (katstevens) [reply dmutils/forms/errors.py]: Yep - up to you @lfdebrux whether you want to include that here

[the remark this answers is not in the document]
Comment (lfdebrux) [reply dmutils/forms/errors.py]: Let's move it in here and and add an import to the previous location.

I'll make the change.

[the remark this answers is not in the document]
Comment (lfdebrux) [reply dmutils/forms/errors.py]: Done

**Comment by lfdebrux** [comment]
@risicle are you happy for me to merge this now?

File: dmutils/__init__.py [hunk 2]
```diff
@@ -2,4 +2,4 @@
 from .flask_init import init_app, init_manager
 
 
-__version__ = '50.2.1'
+__version__ = '50.4.0'
```
Comment (katstevens) [inline dmutils/__init__.py]: What happened to 50.3.0?

**Pull request merged**